### PR TITLE
[MOS-647] Fix AVRCP volume control

### PR DESCRIPTION
**Pull request by Lefucjusz**
**Description**

Fix of the AVRCP volume control.
Minor code cleanup.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible.
- [ ] Has documentation updated

### CODE REVIEW

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.cpp [hunk 1]
```diff
@@ -128,41 +129,41 @@ namespace bluetooth
...
+        if (packetType != HCI_EVENT_PACKET || hci_event_packet_get_type(packet) != HCI_EVENT_AVRCP_META) {
             return;
         }
 
-        status = packet[5];
         if (AVRCP::mediaTracker.avrcp_cid == 0u) {
             return;
         }
 
+        std::uint8_t status = packet[5];
```
Comment (Maciej-Mudita) [inline module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.cpp:149]:
```suggestion
        const std::uint8_t status = packet[5];
```

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp [hunk 2]
```diff
@@ -19,22 +19,14 @@ namespace bluetooth
 {
     class AVRCP
     {
+      private:
+        static std::uint16_t targetVolumeToPercent(std::uint16_t volume);
+        static std::uint16_t controllerVolumeToPercent(std::uint16_t volume);
+
       public:
-        constexpr static const uint8_t subunitInfo[] = {0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3,
+        static constexpr const uint8_t subunitInfo[] = {0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3,
```
Comment (Maciej-Mudita) [inline module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp:27]:
```suggestion
        static constexpr uint8_t subunitInfo[] = {0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3,
```

**Review by Maciej-Mudita — APPROVED** [review]

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.cpp (hunk 1 above)
Comment (Lefucjusz) [reply module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.cpp]: Fixed!

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp (hunk 2 above)
Comment (Lefucjusz) [reply module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp]: Fixed!

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp [hunk 3]
```diff
@@ -19,22 +19,14 @@ namespace bluetooth
...
-        constexpr static uint8_t companiesNum        = 1;
-        constexpr static uint8_t companies[]         = {
-            0x00, 0x19, 0x58 // BT SIG registered CompanyID
-        };
+      private:
+        static std::uint16_t targetVolumeToPercent(std::uint16_t volume);
+        static std::uint16_t controllerVolumeToPercent(std::uint16_t volume);
 
-        constexpr static uint8_t eventsNum     = 6;
-        constexpr static uint8_t events[]      = {AVRCP_NOTIFICATION_EVENT_PLAYBACK_STATUS_CHANGED,
```
Comment (SP2FET) [inline module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp:32]: are those events unnecessary now?

**Review by SP2FET — APPROVED** [review]

File: module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp (hunk 3 above)
Comment (Lefucjusz) [reply module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp:32]: These variables were unused after changes introduced by [this commit](https://github.com/mudita/MuditaOS/commit/9a6b8363351d722f037d4dbe5dd1f1cac25476ac), so I removed them as a part of cleanup. Three of these events are still in use, but the way they're registered has changed - see lines 45-48 of AVRCP.cpp.

**Pull request merged**